compute/admin_guide/install/install_defender/install_serverless_defender.adoc

```diff
@@ -28,9 +28,10 @@ The following runtimes are supported for AWS Lambda:
 * Python 3.6, 3.7, 3.8, 3.9
 * Ruby 2.7
 
-The following runtimes are supported for Azure Functions:
+The following runtimes are supported for Azure Functions (Windows only):
```

Comment:
@avish9 
Did the support was added for manually embedded Defenders on Windows only or also for Vulnerability and Compliance scanning?

Reply:
ping @avish9

Reply:
@avish9  does this change impact this page https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/system_requirements

Reply:
@rcaballeromx can you follow up on this for sys req updates? I am merging this to publish the correct information in this topic.